src/types.ts

```diff
@@ -23,6 +23,16 @@ export interface ExtendedTranslator extends stardust.Translator {
   language(): string;
 }
 
+export interface Galaxy {
```

Comment:
is galaxy a term that we use for this in other places? I don't think we should introduce terms like this in the nebula eco-system unless it is thought through.

Reply:
Yes, https://qlik.dev/libraries-and-tools/nebulajs/in-qlik-sense#exposed-from-qlik-sense and https://github.com/qlik-oss/sn-pivot-table/blob/main/src/index.ts#L15

Reply:
great